File: C/6.c
#include<stdio.h>
#include<conio.h>
void main()
{
    int a,b,q,r;
    clrscr();
    printf("Enter a number :");
    scanf("%d",&a);
    printf("\n Enter the number you want to divide the first number with :");
    scanf("%d",&b);
    r=a%b;
    q=a/b;
    printf("\n The remainder is : %d",r);
    printf("\n The quotient is :%d",q);
    getch();
}

File: PHP/6.php
<?php
/**
 * program to Compute Quotient and Remainder
 * 
 * the slash operator (/) is operator for divide an integer.
 * the modulus operator (%) is operator for get the quotient of 2 value.
 * 
 */
$x = intval(readline("Enter The Number : "));
$div = intval(readline("Enter the divider : "));
$remainder = $x / $div;
$quotient = $x % $div;
echo "the Remainder is : " . $remainder;
echo "\nthe Quotient is : " . $quotient;
?>